community_supplied/Interface/oc-interface-throughput.rule

```diff
@@ -1,11 +1,350 @@
+                            message "$interface-name output bandwidth utilization($out-bw-util%) crossed threshold value ($threshold %). Mbps:$out-mbps Output bytes:$output-bytes";
+                        }
+                    }
+                }
+                /*
+                * Sets color to yellow and sends out an anomaly notification when
+                * the dynamic threshold ($dt-out-bw-util) of output bandwidth
+                * utilization is equal to 1.
+                */
+                term is-output-bandwidth-above-dt {
+                    when {
+                        equal-to "$dt-out-bw-util" 1;
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "$interface-name output bandwidth utilization ($out-bw-util %) is above dynamic threshold. Mbps:$out-mbps Output Bytes:$output-bytes Predicted BW utilization:$predicted-out-bw-util";
+                        }
+                    }
+                }
+                /*
+                * Defaults color to green.
+                */
+                term is-output-bandwidth-normal {
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name output bandwidth utilization ($out-bw-util %) is normal. Mbps:$out-mbps Output bytes:$output-bytes";
+                        }
+                    }
+                }
+            }
+            /*
+            * Variables with default values. Default values can be changed
+            * while deploying playbook instance.
+            */
+            variable in-stats-name {
+                value in-octets;
+                description "Interface input traffic counter name";
+                type string;
+            }
+            variable interface-name-variable {
+                value .*;
+                description "Enter interface name in regex i.e. ge-.*";
+                type string;
+            }
+            variable out-stats-name {
+                value out-octets;
+                description "Interface output traffic counter name";
+                type string;
+            }
+            variable speed {
+                value high-speed;
+                description "Interface speed state name";
+                type string;
+            }
+            variable threshold {
+                value 75.00;
+                description "Interface bandwidth percent high threshold value in %.";
+                type float;
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                supported-healthbot-version 3.1.0;
+                supported-devices {
+                    juniper {
+                        operating-system junos {
+                            products EX {
+                                releases 17.3R1 {
```

Comment:
I don't think EX. supports OC

Reply:
EX supports a particular platform in OC, so i made the required changes